### PR TITLE
Add script for installing hypershift clusters using rosa cli

### PR DESCRIPTION
**Pull request by morenod**

### CODE REVIEW

**Review by dry923 — APPROVED** [review]
LGTM!

File: rosa-hypershift/rosa-hosted-wrapper.py [hunk 1]
```diff
@@ -0,0 +1,970 @@
...
+    os.chdir(my_path + '/e2e-benchmarking/workloads/kube-burner')
+    load_env["JOB_ITERATIONS"] = str(jobs)
+    load_env["CHURN"] = "true"
+    load_env["CHURN_DURATION"] = load_duration
+    load_env["CHURN_PERCENT"] = "10"
+    load_env["CHURN_WAIT"] = "30s"
+    load_env["JOB_TIMEOUT"] = "6h"
+    load_env["CLEANUP_WHEN_FINISH"] = "true"
+    load_env["INDEXING"] = "true"
+    load_env["HYPERSHIFT"] = "true"
```
Comment (mukrishn) [inline rosa-hypershift/rosa-hosted-wrapper.py:373]: @morenod how does the script pass [these two](https://github.com/cloud-bulldozer/perfscale-managed-services/blob/main/hypershift/hosted-wrapper.py#L260-L261) env variables to the e2e workload ?

Comment (morenod) [reply rosa-hypershift/rosa-hosted-wrapper.py:373]: Good catch. just added

File: rosa-hypershift/rosa-hosted-wrapper.py [hunk 2]
```diff
@@ -0,0 +1,972 @@
...
+    load_env["MGMT_CLUSTER_NAME"] = mgmt_cluster_name
+    load_env["HOSTED_CLUSTER_NS"] = "clusters-" + hosted_cluster_name
```
Comment (mukrishn) [inline rosa-hypershift/rosa-hosted-wrapper.py]: Hosted Cluster namespace format is different now,  
```
ocm-staging-216r2b3oor0hlrppdmffa0sd1sfgq7j7-mrnd-fr3-0001
```
so lets do regex formats `load_env["HOSTED_CLUSTER_NS"] = ".*-" + hosted_cluster_name`

**Review by mukrishn — APPROVED** [review]
LGTM

**Pull request merged**